### PR TITLE
Add a Github Action for testing changes, closes #36

### LINKED ISSUE #36: Migrate to GitHub Actions
As titled

### PR DESCRIPTION
**Pull request by abskmj**

### CODE REVIEW

**Comment by abskmj** [comment]
This PR adds a Github action to run the test cases when new code is pushed on any branch. It is similar to how Travis CI is setup currently on this repository. It runs test cases on Node v10 & v12.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    strategy:
+      matrix:
+        node: [ '10', '12' ]
+    name: Test with Node v${{ matrix.node }} 
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - run: npm install
```
Comment (Eomm) [inline .github/workflows/test.yml]:
```suggestion
      - run: npm install --ignore-scripts
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: Test
```
Comment (Eomm) [inline .github/workflows/test.yml:1]: Could you add the build status badge in the README and delete the travis.yml file?

Comment (abskmj) [reply .github/workflows/test.yml:1]: Updated

**Review by Eomm — APPROVED** [review]

**Pull request merged**